### DISCUSSION TITLE
Hires checkpoint

### DISCUSSION THREAD

**Discussion by ianbrophy** [vladmandic/sdnext#2289] [Q&A] [answered]
Hello, I recently installed SD.Next and I'm wondering, is there a way to set a separate checkpoint for hires/second pass?

**Comment by vladmandic** [comment, marked as answer]
second pass = optional upscale +optional hires + optional refiner.
hires uses selected base model
refiner uses selected refiner model (which can be anything)

knowing that, you can mix and match

**Reply by ianbrophy** [reply]
I will give that a go! Thank you.
